e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -1,16 +1,17 @@
-import _ from "underscore";
 import {
   createAction,
   describeEE,
   describeWithSnowplow,
   enableTracking,
   expectGoodSnowplowEvents,
   expectNoBadSnowplowEvents,
+  modal,
```

Comment:
When I edit an implicit action I don't appear as the last editor (but I do appear as its creator). Looks like a bug (perhaps not related to this PR).

https://github.com/metabase/metabase/assets/6830683/545c8000-a679-4b85-89a2-31540b3434b1

Reply:
Weird, though I'm not sure there's much I can do about this one. Maybe @qnkhuat can chime in on this one?

Reply:
I can't watch the recorded video

Reply:
> I can't watch the recorded video

@qnkhuat Did you try downloading it, or playing in a different browser (not Firefox)?

Reply:
ah good catch. the answer is we don't support filtering by editor for actions because we haven't recorded revisions for actions.

I actually made a note here https://github.com/metabase/metabase/blob/global-search-more-filter-options/src/metabase/search/filter.clj#L218